ros/angel_utils/demo_ui/.gitignore

```diff
@@ -0,0 +1 @@
+/node_modules/
```

Comment:
the `npm install` also creates `log`, `build`, and `install` directories that can be ignored

Reply:
Where do these get generated? I don't see them in my space.

Reply:
hmmm, I see them under demo_ui/ but maybe it's from something else